packages/web/pages/_app.tsx

```diff
@@ -72,3 +70,9 @@ function checkH1Count() {
     }
   }, 500)
 }
+
+function hashScroller(id: string) {
```

Comment:
very very very small nit but i think it's a little easier to parse when functions are above the component

Reply:
ah good to know, ive found im pretty indifferent with a 1% lean towards bottom but i do find having all below or all above seems nicest.